.changeset/tiny-lions-stare.md

```diff
@@ -0,0 +1,5 @@
+---
+"@navikt/ds-react": minor
+---
+
+ToggleGroup: Supports `icon` and `label`-prop as a replacement for `children`. Children is marked as deprecated and will be removed in future major-versions.
```

Comment:
```suggestion
ToggleGroup: Added props `icon` and `label` as a replacement for `children`. Children is marked as deprecated and will be removed in a future major-version.
```
Just a suggestion. I think it sounds better, but it might just be me 😅